hydra_agent/tests/test_agent.py

```diff
@@ -209,9 +225,9 @@ def test_post(self, put_session_mock, post_session_mock,
         get_new_object = self.agent.get(new_object_url)
         self.assertEqual(get_new_object, new_object)
 
-    @patch('hydra_agent.agent.Session.get')
-    @patch('hydra_agent.agent.Session.delete')
-    @patch('hydra_agent.agent.Session.put')
+    @ patch('hydra_agent.agent.Session.get')
+    @ patch('hydra_agent.agent.Session.delete')
```

Comment:
Why do we have space here? `@ patch`

Reply:
Oh! Must be the formatter issue. It was completely unintentional.